pkgs/data/fonts/whatsapp-emoji/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenvNoCC
+, lib
+, fetchFromGitHub
+, imagemagick
+, nix-update-script
+, pngquant
+, python3Packages
+, which
+, zopfli
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "whatsapp-emoji-linux";
+  version = "2.22.8.79-1";
+
+  src = fetchFromGitHub {
+    rev = "refs/tags/2.22.8.79-1";
```

Comment:
why don't you parameterize version here?

Reply:
done

Reply:
What was done?

Reply:
Version parametrization and pname deparametrization